english/dream/lucidDream.md

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Caroline, 

i just committed the change to my own branch in my fork.  i used git rm lucidDream.md to remove it.  (we didn't do that in the course, did we?)

i see that lucidDream.md is not in 
either my branch or the master branch of my fork and i wouldn't expect it to have gone to the original repository. 

was it removed just because of my commit? or did you or someone else remove it.  I was expecting the need to submit another pull request, but now it doesnt seem necessary.

Thanks. 
Bill



Sent from my iPod

On Apr 1, 2015, at 4:25 PM, Caroline Buckey <notifications@github.com> wrote:

> In english/dream/lucidDream.md:
> 
> > @@ -0,0 +1,15 @@
> > +
> It looks like maybe you meant this file to be replaced by the other lucid dream file? If so, please remove this file from the pull request by committing a change that removes the file.
> 
> —
> Reply to this email directly or view it on GitHub.
>